docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+
+### Open Questions
+ 3.  Can we punt on adding users as organization members?  Still don't see a clear answer on this.
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
+ 4.  Can we punt on project transfering?  Sounds like yes.  A cluster-admin could transfer ownership.
+
+     An org-owner shouldn't be able to take a project from someone else and they shouldn't be allowed to orphan their own project.  That means we'd need an offer/request/approval flow.
+
+ 6.  What things can org-owners do?
+
+ 	There's a list below proposing things.
+
+
+## ClusterResourceQuota
+`ClusterResourceQuota` mirrors `ResourceQuota`, but at the cluster-scope.  (We can't call it that because `ResourceQuota` is a namespace-scope resource and it can't be in both scope.)  A `ClusterResourceQuota` has an optional owner that is either a user or an organization and an optional label selector.  An empty label selector is `true`.  If both are set, then the `ClusterResourceQuota` applies to all namespaces matching the selector **and** owned by the owner.
+
+The existing quota admission controller would remain in place, but no limits would be set by default.  If a project owner decided that a particular project was starving the rest, he could assign hard limits to that Project's `ResourceQuota` to prevent it from taking more.  An additional admission controller would be responsible for doing the rollup across all projects in the Organization and preventing over-use of resources.
+
+```go
+type ClusterResourceQuota struct {
+	kapi.TypeMeta
+	kapi.ObjectMeta
+
+	// Spec defines the desired quota
+	Spec ResourceQuotaSpec 
+
+	// Status defines the actual enforced quota and its current usage
+	Status ResourceQuotaStatus 
+}
+
+type ClusterResourceQuotaSpec struct {
+	Owner    *kapi.ObjectReference
+	Selector map[string]string 
+
+	kapi.ResourceQuotaSpec
+}
+```
+
+#### Limitations
+This approach is relatively easy to build, but it is limited because it doesn't reserve quota for a given project.  A more complicated approach would be to allow killing of "extra" resources and then provide a rate and ceiling like tc.  This would have an overall cap and allow "borrowing" of quota when resources are plentiful, but then selectively kill resources when resources are scarce to allow for project guarantees.  With enough effort, this sort of solution could be layered on top of kube by setting the ceiling for the existing quota controller and having a separate resource for the guarantee and a separate controller for the killer.
+
+I'm sure there will be complaints no matter what we do.
```

Comment:
I would like to keep things simple, so I agree with the Limitations for now.  If we need a balancer that can adjust P1 RQ.Spec.Hard[cpu] lower and give more to P2 RQ.Spec.Hard[cpu] because it sees its closer to reaching its threshold by looking at RQ.Spec.Used[cpu] those types of policy adjustments can be layered on top.